recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set python_min = '3.12' %}
+{% set python_min = '3.9' %}
```

Comment:
3.9 is conda-forge's default. You only need to set that variable if your min is higher than that.

```suggestion
```